### DISCUSSION TITLE
add tailwind into this template

### DISCUSSION THREAD

**Discussion by yaffalhakim1** [shuding/nextra-docs-template#14] [Q&A] [answered]
is it possible to add tailwind into this template?
since this doesn't have globals.css where do i put @tailwind base ?

**Comment by yaffalhakim1** [comment, marked as answer]
I found how to this.

1.  install tailwind for nextjs like usual from the tailwind docs
2. add `mdx` and `theme.config.tsx` like this in your tailwind config
` content: [
    "./app/**/*.{js,ts,jsx,tsx,mdx}",
    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
    "./components/**/*.{js,ts,jsx,tsx,mdx}",
    "./theme.config.tsx",

    // Or if using `src` directory:
    "./src/**/*.{js,ts,jsx,tsx, mdx}",
  ],`
3. add the @ tailwindbase etc into a file that you created with name styles.css
4. import it in _app.mdx, create one if you don't have _app.mdx file
5. done

**Comment by GuiltyRemnant** [comment]
Thanks for providing that how-to, but when you create an _app.mdx file, there is no nextra anymore, just a blank page. And when you import the globals.css to your index.mdx, it says:
![image](https://github.com/shuding/nextra-docs-template/assets/23167815/7369aaa6-a6d2-429b-b338-2b9863968425)

**Reply by ninjasweb** [reply]
Create a _app.js file in pages folder like this:

import styles from "./styles.css"

export default function App({ Component, pageProps }) {
  return (
    <div className={styles.app}>
      <Component {...pageProps} />
    </div>
  )
}

This solve my problem
